discord-bot/src/commands/meeting.ts

```diff
@@ -0,0 +1,67 @@
+import { CommandInteraction } from 'discord.js';
+import { ApplicationCommandOptionType } from 'discord.js';
+import { Discord, Slash, SlashOption } from 'discordx';
+import { prisma } from '../config/db.js';
+import { env } from '../config/env.js';
+
+@Discord()
+export class Meeting {
+    @Slash({ description: "Meeting" })
+    meeting(
+        @SlashOption({name: "subject", description: "subject of the meeting", required: true, type: ApplicationCommandOptionType.String})
+        subject: string,
+
+        @SlashOption({name: "year", description: "year of the meeting", required: true, type: ApplicationCommandOptionType.Integer})
+        year: number,
+
+        @SlashOption({name: "month", description: "month of the meeting", required: true, type: ApplicationCommandOptionType.Integer})
+        month: number,
+
+        @SlashOption({name: "day", description: "day of the meeting", required: true, type: ApplicationCommandOptionType.Integer})
+        day: number,
+
+        @SlashOption({name: "hour", description: "hour of the meeting", required: true, type: ApplicationCommandOptionType.Integer})
+        hour: number,
+
+        @SlashOption({ name: "minute", description: "minute of the meeting", required: false, type: ApplicationCommandOptionType.Integer})
+        minute: number,
+
+        interaction: CommandInteraction
+    ): void {
+        var webhook = prisma.discordBotWebhook.findUnique({
```

Comment:
```suggestion
        let webhook = prisma.discordBotWebhook.findUnique({
```
Use `let `over `var`

Reply:
Everything's ok otherwise, just remember to clean the nested promises sometimes